spec/labor/coupon_generator_spec.rb

```diff
@@ -5,10 +5,8 @@
 
   describe "#expect" do
     it "generates code with proper prefix ( [version]_[code] )" do
-      VCR.use_cassette "coupon_generator_1" do
-        version = versions.sample
-        expect(described_class.new(1, version).generate).to include("#{version}_")
-      end
+      version = versions.sample
```

Comment:
I think you can delete `spec/support/fixtures/vcr_cassettes/coupon_generator_1.yml` if it's not needed